book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(person, date)
+    Rental.new(date, person, self)
+  end
+end
```

Comment:
- [ ] Ouch! You're doing top-notch work so far but it seems like you forgot to add the code for the "A person has many rentals" relationship in the `Person` class as specified in the project requirements. 
- [ ] So please make sure to add an instance variable `rentals` (with an accessor) and initialize it to an empty array in the constructor and then define an `add_rental` method that accepts a `book` & `date` which will then create a rental similar to the one at line `11` in the above-highlighted code. (This should be added in the Person class)